### PR TITLE
Sapphire graph

### PR DESCRIPTION
**Pull request by lubej**
## Solution

Update Home page graph, so it includes Sapphire support.

## Links

https://www.loom.com/share/88e7efa1a0474c499eb9080cdd875979

### CODE REVIEW

**Review by csillag — APPROVED** [review]
Reviewed. LGTM.

File: src/app/pages/HomePage/Graph/Graph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,641 @@
...
+            cy="295.904"
+            r="8.02878"
+            transform="rotate(-90 155.763 295.904)"
+            fill="#191932"
+          />
+          <text x="152.5" y="297" fill="#fff" fontSize="4px">
+            {t('home.txn')}
+          </text>
+        </Link>
+        <Link to={RouteUtils.getLatestBlocksRoute(Layer.Emerald)}>
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/Graph/index.tsx:309]: Was reviewed so not looking at code, but here we should have `Sapphire`

Comment (csillag) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:309]: Yes, I was planning to fix/replace all the links in the subsequent PRs, when we actually build those pages.

Comment (lubej) [reply src/app/pages/HomePage/Graph/Graph/index.tsx]: Nice catch

**Pull request merged**